src/hooks/useNotificationTypes.tsx

```diff
@@ -232,6 +234,11 @@ export const notificationTypes: NotificationTypeConfig[] = [
             },
           });
 
+          track(AnalyticsEvent.TransferNotification, {
```

Comment:
when does this get emitted? when there are multiple transfers, will this get emitted again even though it's finished?

Reply:
yeah this actually will get emitted way too many times. when we poll the notification will rerender/retrigger. i mistakenly believed that it wouldn't happen due to react caching w/ usecallback and useeffect dependency arrays but this will spam amplitude. going to handle this in the api side